dtests/src/test/java/io/snappydata/hydra/cluster/SnappyTest.java

```diff
@@ -170,7 +209,7 @@ protected void generateSnappyConfig() {
                 SnappyBB.getBB().getSharedMap().put("locatorPort", portString);
                 Log.getLogWriter().info("Generated locator endpoint: " + endpoint);
                 SnappyNetworkServerBB.getBB().getSharedMap().put("locator" + "_" + RemoteTestModule.getMyVmid(), endpoint);
-            } else if (dirPath.contains("Store") || dirPath.contains("server")) {
+            } else if (dirPath.contains("Store") || dirPath.contains("server") || dirPath.contains("store") || dirPath.contains("accessor")) {
```

Comment:
use dirPath.toLowerCase().contains("store") instead of two checks and same could be used for accessors and server